manifests/tftp/netboot.pp

```diff
@@ -57,12 +62,18 @@
         unless  => "/bin/grep -q regexp '${root}/grub2/grubx64.efi'",
         require => Package[$packages],
       }
+
```

Comment:
Please don't do this. The next line is dependent on this line and the fact that there's no empty line helps to indicate that. Note the `->` on the next line.